packages/react-i18n/package.json

```diff
@@ -51,7 +51,7 @@
   },
   "sideEffects": false,
   "peerDependencies": {
-    "react": "^16.0.0"
+    "react": ">= 16.0.0"
```

Comment:
I think if we lock in the ranges we should be fine. Web usually is the main focus of what we use and support. Re: https://github.com/Shopify/quilt/issues/1740